packages/@ama-sdk/core/src/fwk/date.spec.ts

```diff
@@ -94,6 +95,7 @@ describe('Date', () => {
 
   it('should support json conversions', () => {
     const originalDate = new utils.Date(new Date());
+    // eslint-disable-next-line unicorn/prefer-structured-clone -- we test the JSON conversion
```

Comment:
maybe we should have a different eslint config for test files ?

Reply:
Maybe but not for this use case
I think it's a particular case we want explicitly do JSON conversion
even in test we can use structuredClone instead of cloning with JSON conversion